cryptography/src/bls12381/dkg/ops.rs

```diff
@@ -95,16 +95,16 @@ pub fn verify_share<V: Variant>(
 
 /// Construct a public polynomial by summing a vector of commitments.
 pub fn construct_public<V: Variant>(
-    commitments: Vec<poly::Public<V>>,
+    commitments: &BTreeMap<u32, poly::Public<V>>,
```

Comment:
While we don't use the dealer identities, this interface makes the calling code very clean.